### PR TITLE
4326 doc examples&replace fixes

### PR DESCRIPTION
**Pull request by mheppler**
## Related Issues

- connects to #4326 Guides - Review the use of examples

## Pull Request Checklist

- [ ] Unit [tests][] completed
- [ ] Integration [tests][]: None
- [ ] Deployment requirements, [SQL updates][], [Solr updates][], etc.: None
- [x] [Documentation][docs] completed
- [x] Merged latest from "develop" [branch][] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: https://github.com/IQSS/dataverse/tree/develop/scripts/database/upgrades
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/7.2.1/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

File: doc/sphinx-guides/source/user/dataset-management.rst [hunk 1]
```diff
@@ -198,11 +196,11 @@ File tags are comprised of custom, category (i.e. Documentation, Data, Code) and
...
 
-After successfully replacing a file, a new dataset draft version will be created. A summary of your actions will be recorded in the dataset Version Details table in the "Versions" tab on that dataset's page, and in the file Version Details table on the file's page. Both tables allow you to access all previous versions of the file across all previous versions of your dataset, including the old version of the file before you replaced it.
+After successfully replacing a file, a new dataset draft version will be created. A summary of your actions will be recorded in the "Versions" tab on that dataset's page, and in "Versions" tab on the file's page. Both tabs allow you to access all previous versions of the file across all previous versions of your dataset, including the old version of the file before you replaced it.
```
Comment (mheppler) [inline doc/sphinx-guides/source/user/dataset-management.rst]: Suggested revision...
> A summary of your actions will be recorded in the "Versions" tab on both the dataset page and file page.

File: doc/sphinx-guides/source/user/find-use-data.rst [hunk 2]
```diff
@@ -10,7 +10,7 @@ Finding Data
 Without logging in to Dataverse, users can browse Dataverse, search for dataverses, datasets, and files, view dataset descriptions and files for
 published datasets, and subset, analyze, and visualize data for published (restricted & not restricted) data files. To view an unpublished dataverse, dataset, or file, a user will need to be given permission from that dataverse's administrator to access it.
 
-A user can search the dataverses, datasets, and files within a particular dataverse by using the search bar found on a dataverse page. For example, if you are on the Murray Research Archive Dataverse page, you can search that specific dataverse's contents by using the search bar and/or facets displayed on the page.
+A user can search within a specific dataverse for the dataverses, datasets, and files it contains by using the search bar and facets found on that dataverse's page.
```
Comment (mheppler) [inline doc/sphinx-guides/source/user/find-use-data.rst]: "...found on that dataverse's page" looked odd to me. Suggested revision...
> A user can use the search bar and facets to discover the dataverses, datasets, and files contained within a specific dataverse.

File: doc/sphinx-guides/source/api/native-api.rst [hunk 3]
```diff
@@ -339,7 +339,7 @@ A more detailed "add" example using curl::
 
 Example python code to add a file. This may be run by changing these parameters in the sample code:
 
-* ``dataverse_server`` - e.g. https://dataverse.harvard.edu
+* ``dataverse_server`` - e.g. https://dataverse.example.edu
```
Comment (mheppler) [inline doc/sphinx-guides/source/api/native-api.rst]: Would it be too much effort to use `example.dataverse.org` as an example, and then setup a redirect for that example domain to demo.dataverse.org? @pdurbin...??

**Comment by pdurbin** [comment]
Please no ampersands in branch names. I can work around this easily but check this out:

```
$ git checkout 4326-doc-examples&replace-fixes
[1] 98185
-bash: replace-fixes: command not found
murphy:guides.dataverse.org pdurbin$ error: pathspec '4326-doc-examples' did not match any file(s) known to git.

[1]+  Exit 1                  git checkout 4326-doc-examples
```

Thanks.

File: doc/sphinx-guides/source/installation/config.rst [hunk 4]
```diff
@@ -695,7 +695,7 @@ See :ref:`Branding Your Installation` above.
 
 By default the footer says "Copyright © [YYYY]" but you can add text after the year, as in the example below.
 
-``curl -X PUT -d ", The President &#38; Fellows of Harvard College" http://localhost:8080/api/admin/settings/:FooterCopyright``
+``curl -X PUT -d ", Foobar College" http://localhost:8080/api/admin/settings/:FooterCopyright``
```
Comment (pdurbin) [inline doc/sphinx-guides/source/installation/config.rst]: Foobar College?

File: doc/sphinx-guides/source/user/find-use-data.rst [hunk 5]
```diff
@@ -10,7 +10,7 @@ Finding Data
 Without logging in to Dataverse, users can browse Dataverse, search for dataverses, datasets, and files, view dataset descriptions and files for
 published datasets, and subset, analyze, and visualize data for published (restricted & not restricted) data files. To view an unpublished dataverse, dataset, or file, a user will need to be given permission from that dataverse's administrator to access it.
 
-A user can search the dataverses, datasets, and files within a particular dataverse by using the search bar found on a dataverse page. For example, if you are on the Murray Research Archive Dataverse page, you can search that specific dataverse's contents by using the search bar and/or facets displayed on the page.
+A user can search within a specific dataverse for the dataverses, datasets, and files it contains by using the search bar and facets displayed on that dataverse's page.
```
Comment (pdurbin) [inline doc/sphinx-guides/source/user/find-use-data.rst]: I'm glad this MRA example was taken out. Nothing against the archive, it's just most people haven't heard of it.

**Pull request merged**